scripts/update_python.py

```diff
@@ -108,7 +108,7 @@ def build_python():
         check_call(['brew', 'install', 'openssl', 'pkg-config'])
         if platform.machine() == 'x86_64':
             prefix = '/usr/local'
-            min_macos_version = '10.13'
+            min_macos_version = '10.11'
```

Comment:
I guess we need to upload a new binary package?  If so, we should bump `revision` above so that filename is different.